config/initializers/devise.rb

```diff
@@ -275,9 +275,9 @@
   # If you want to use other strategies, that are not supported by Devise, or
   # change the failure app, you can configure them inside the config.warden block.
   #
-  config.warden do |manager|
-    manager.failure_app = DeviseFailureApp
-  end
+  # config.warden do |manager|
+  #   manager.failure_app = DeviseFailureApp
+  # end
```

Comment:
Please add this back in for the iOS app.

Reply:
Done. Reverted changes back.